project/project.go

```diff
@@ -10,9 +10,19 @@ type Project struct {
 }
 
 type Config struct {
-	Code          string       `json:"code"`
-	CodeExpParam  string       `json:"codeExpParam,omitempty"`
-	VMType        types.VM     `json:"vmType"`
-	OutputType    types.Output `json:"outputType"`
-	OutputAddress string       `json:"outputAddress,omitempty"`
+	Code         string   `json:"code"`
+	CodeExpParam string   `json:"codeExpParam,omitempty"`
+	VMType       types.VM `json:"vmType"`
+	Output       struct {
```

Comment:
maybe define a OutputConfig struct would be cleaner

Reply:
> maybe define a OutputConfig struct would be cleaner

Okay

Reply:
> and also, the project config file in test/project dir need changed too

Actually `stdout` is used by default if output not configured, but we can also explicitly configure output as stdout